site/content/admin-guide/limits-config.md

```diff
@@ -0,0 +1,13 @@
++++
+title = "Limits Configuration"
+weight = 325
++++
+
+To enable the limits based on the default prometheus based implementation, the parameters in the below table should be configured.
+
+| Environment Variable<br>Command Line Option | Mandatory | Default Value | Description  |
+| :------------------------------------------ | :-------: | :------------ | :------------|
+| `hono.plan.prometheusBased.host` | no | none | The host name or ip of the prometheus backend which will be used by the default implementation to retrieve metrics data.|
+| `hono.plan.prometheusBased.port` | no | 9090 | The port of the prometheus backend which will be used by the default implementation to retrieve metrics data.|
+
```

Comment:
this needs to go into the admin guides of the adapters that support it.

Reply:
Done.